.github/workflows/upgrade-test.yml

```diff
@@ -224,6 +224,8 @@ jobs:
           chmod +x ${{ env.BINARY_ROOT_PATH }}/engine-runner
           echo "/usr/lib before copy of .so files"
           ls -l /usr/lib
+          # TODO: Remove this once we have implemented fetching the shared libs in the engine-runner build script
+          curl https://repo.chainflip.io/lib/1_4_6/libchainflip_engine_v1_4_6.so -o /usr/lib/libchainflip_engine_v1_4_6.so
```

Comment:
this should come from rebasing, why picked here? even though we squash merge, just pollutes the git history a little